mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/task/SolveMessageInconsistenciesServiceTest.java

```diff
@@ -149,6 +154,65 @@ void fixMessageInconsistenciesShouldReturnCompletedWhenInconsistentData() {
                 .isEqualTo(Task.Result.COMPLETED);
         }
 
+        @Test
+        void outdatedImapUidDAOEntryShouldBeConfirmed(CassandraCluster cassandra) throws Exception {
```

Comment:
I'm trying to understand the test: you want to ensure that when you are running a "fix message inconsistencies" job and the job is detecting a message has been inconsistent because it saw a partial write, it's finally not treated as such because you check the state of the message twice. That's it?

Reply:
That's it?

Reply:
what about: shouldNotConciderPendingMessageWriteAsInconsistency?